.drone.yml

```diff
@@ -8,5 +8,21 @@ steps:
     - export LC_ALL=C.UTF-8
     - apt-get update -y
     - apt-get install -y ghc cabal-install
+    - cabal --version
     - cabal update
     - cabal new-test
+---
+kind: pipeline
+name: i386
+platform: { os: linux, arch: amd64 }
+steps:
+- name: Test
+  image: i386/ubuntu
+  commands:
+    - export LC_ALL=C.UTF-8
+    - apt-get update -y
+    - apt-get install -y ghc cabal-install
+    - cabal --version # 1.24
+    - cabal update
+    - cabal install --dependencies-only --enable-tests --constraint 'optparse-applicative -process'
```

Comment:
I didn't know that you could specify flags in this constraint syntax. Neat!

Reply:
The flag is needed to break a circular dependency through `optparse-applicative` / `process` / `unix` / `bytestring`. It seems that newer versions of Cabal solver are keen to supply `-process` flag automatically, so we do not usually need to set it manually, but old Cabals get stuck.

Reply:
I think that might be worth a brief comment in the code. LGTM in any case.